src/fair/allocation.py

```diff
@@ -24,39 +31,71 @@ def initialize_allocation_matrix(items, agents):
     return X
 
 
-def initialize_exchange_graph(items):
+def initialize_edge_matrix(N: int):
+    """Initialize array that keeps track of the agents responsible for a specific edge between two
+    different items on the exchange graph.
+    Initially, there are no edges, thus edge matrix is a len(items) x len(items) array of empty lists.
+
+    Args:
+       N (int): Number of items
+
+    Returns:
+        list: N x N list of empty lists
+    """
```

Comment:
I think you could do this with a one-liner:

```
[[[] for i in range(N)] for j in range(N)]
```

Reply:
Changed (eliminated the function altogether)